kratos/utilities/read_materials_utility.cpp

```diff
@@ -169,15 +169,15 @@ void ReadMaterialsUtility::AssignMaterialToProperty(
     )
 {
     KRATOS_TRY;
-    
-    // Assign CL
-    AssignConstitutiveLawToProperty(MaterialData, rProperty);
 
     // Assign variables
     AssignVariablesToProperty(MaterialData, rProperty);
     
     // Assign tables
     AssignTablesToProperty(MaterialData, rProperty);
+    
+    // Assign CL
+    AssignConstitutiveLawToProperty(MaterialData, rProperty);
```

Comment:
is this change still necessary?

Otherwise looks good to me :+1:

Reply:
If we want to avoid some potential errors we should

Reply:
what error?
I am hesitant modifying such an important class without a good reason

Reply:
Basically there are checks that are trigered when reading the variables if the order of the reading is done in the current manner, changing the order solves the issue and preserves the current behaviour (because right now there isn't a CL trying to deduce any internal variable from not registered variables)

Reply:
I am favorable to this change, even though i think it is actually not needed any longer